<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OceanDataViewOptionsDialog</class>
 <widget class="QDialog" name="OceanDataViewOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ocean Data View options</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="InputFile_groupBox">
     <property name="title">
      <string>Input file</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Encoding:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="CodecInput_ComboBox">
        <item>
         <property name="text">
          <string>UTF-8</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Apple Roman</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Latin-1 (ISO 8859-1)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="setGeocodeRange_checkBox">
        <property name="text">
         <string>set geocode range</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="setGearID_checkBox">
        <property name="text">
         <string>set gear ID</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="usePANGAEAQualityFlags_checkBox">
        <property name="text">
         <string>use PANGAEA quality flags</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="addLineNo_checkBox">
        <property name="text">
         <string>created a virtual station label by adding the line number</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>Select parameters dialog</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QCheckBox" name="ShowShortName_checkBox">
        <property name="text">
         <string>show parameter short name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="ShowMethod_checkBox">
        <property name="text">
         <string>show method</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="ShowComment_checkBox">
        <property name="text">
         <string>show comment</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Device type</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QRadioButton" name="ODV_DeviceTypeB_radioButton">
        <property name="text">
         <string>set type to B (marker will be displayed)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="ODV_DeviceTypeC_radioButton">
        <property name="text">
         <string>set type to C (no marker will be displayed)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="ODV_DeviceTypeAuto_radioButton">
        <property name="text">
         <string>set type automatically</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string>Output file</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLineEdit" name="OceanDataViewFilename_lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="BrowseOceanDataViewFilename_pushButton">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Program</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="OceanDataViewProgram_lineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="BrowseOceanDataViewProgram_pushButton">
        <property name="text">
         <string>Browse ...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="StartOceanDataView_checkBox">
        <property name="locale">
         <locale language="English" country="UnitedStates"/>
        </property>
        <property name="text">
         <string>start Ocean Data View</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OK_pushButton">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cancel_pushButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>CodecInput_ComboBox</tabstop>
  <tabstop>setGeocodeRange_checkBox</tabstop>
  <tabstop>usePANGAEAQualityFlags_checkBox</tabstop>
  <tabstop>addLineNo_checkBox</tabstop>
  <tabstop>ShowShortName_checkBox</tabstop>
  <tabstop>ShowMethod_checkBox</tabstop>
  <tabstop>ShowComment_checkBox</tabstop>
  <tabstop>ODV_DeviceTypeB_radioButton</tabstop>
  <tabstop>ODV_DeviceTypeC_radioButton</tabstop>
  <tabstop>ODV_DeviceTypeAuto_radioButton</tabstop>
  <tabstop>OceanDataViewFilename_lineEdit</tabstop>
  <tabstop>BrowseOceanDataViewFilename_pushButton</tabstop>
  <tabstop>OceanDataViewProgram_lineEdit</tabstop>
  <tabstop>BrowseOceanDataViewProgram_pushButton</tabstop>
  <tabstop>StartOceanDataView_checkBox</tabstop>
  <tabstop>OK_pushButton</tabstop>
  <tabstop>Cancel_pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
